pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -856,4 +856,28 @@ rec {
     readmeFile = "README_hr_HR.txt";
     license = with lib.licenses; [ gpl2Only lgpl21Only mpl11 ];
   };
+
+  /* TOKI PONA */
+
+  tok = mkDict rec {
+    pname = "hunspell-dict-tok";
+    version = "20220829";
+    dictFileName = "tok";
+    readmeFile = "README.en.adoc";
+
+    src = fetchzip {
+      url = "https://github.com/somasis/${pname}/releases/download/${version}/${pname}-${version}.tar.gz";
```

Comment:
```suggestion
      url = "https://github.com/somasis/hunspell-dict-tok/releases/download/${version}/hunspell-dict-tok-${version}.tar.gz";
```

Reply:
Apparently I didn't even bother to test these changes to the metadata after I made them originally! The GitHub repo name is `hunspell-tok`, not `hunspell-dict-tok`, whoops...